src/stk/molecular/topology_graphs/cage/metal_topologies/m4l6_tetrahedron.py

```diff
@@ -20,6 +20,11 @@ class M4L6Tetrahedron(Cage):
     Building blocks with three functional groups are required for this
     topology.
 
+    When using a :class:`dict` for the `building_blocks` parameter,
+    as in :ref:`multi-building-block-cage-construction`:
+    *Multi-Building Block Cage Construction*, a :class:`.BuildingBlock`
+    needs to be assigned to each of the following vertex ids:
```

Comment:
This one was missed looks like.

Reply:
Lines 20 and 21 give the required information in this case as all vertices are the same.

Reply:
ah yeah, i guess we wanna remove this paragraph then?

Reply:
mmm, either remove or I can see the benefit of it looking the same as the rest but with only one line - thoughts?

Reply:
Yes, I can also see the benefit in that, in which case I guess the paragraph above it would be edited. Certainly from the point of view of someone quickly scanning the documentation, I think that this option is probably better, but I don't have a strong opinion and will leave it to you to decide.